examples/with-fauna/scripts/setup.js

```diff
@@ -190,6 +190,8 @@ const main = async () => {
   console.log('- Created .env.local file with secret')
 }
 
+exports.resolveDbDomain = resolveDbDomain
```

Comment:
It might be better to move this to it's own constants file so that we don't have to import from the setup file.